roles/cifmw_cephadm/tasks/ceph_upgrade.yml

```diff
@@ -0,0 +1,84 @@
+- name: Fail if target image is not available
+  ansible.builtin.fail:
+    msg: "Ceph target image is not available"
+  when: cifmw_cephadm_update_container_image is undefined or
```

Comment:
this two variables are defined always

Reply:
correct, we don't need them: we have defaults or the user is supposed to provide an override. If the image is not valid, cephadm itself will fail checking for `needs_update` using the provided container image.